### ISSUE TITLE
PAYPAL/PAYMILL Problem

### ISSUE THREAD

**Issue by koos** [railslove/startups-fuer-fortuna.github.io#18]
Hier ist es so, dass ich nach einem Paypal Checkout zurück zur App komme aber keinen Callback habe um das Donation-Objekt anzulegen. Bei Kreditkartenzahlung mache ich ja die Transaction im donations_conttroller/donation-model und lege mir da alles zurecht. Bei Paypal-Zahlung wird der User ja direkt zu Paypal redirected.

@LARS: hast Du ‘ne Idee und könntest die reinhacken bitte? Ansonsten muss ich die dann manuell anlegen - geht auch.

**Comment by kangguru** [comment]
kommt alles auf der return URL an: https://developers.paymill.com/guides/paypal/transactions.html#transaction-results

**Comment by koos** [comment]
Könnte das jemand implementieren bitte?

Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pi&cv=6.4.27&pv=8.3]
On Mo., Juli 6, 2015 at 9:02 vorm., Lars <notifications@github.com> wrote:
kommt alles auf der return URL an: 
https://developers.paymill.com/guides/paypal/transactions.html#transaction-results
[https://developers.paymill.com/guides/paypal/transactions.html#transaction-results]

—
Reply to this email directly or view it on GitHub
[https://github.com/railslove/startups-fuer-fortuna.github.io/issues/18#issuecomment-118747273] .[https://github.com/notifications/beacon/AAAgl7nyeiW9S_6_eVSAw-sT67Wtesukks5oah-igaJpZM4FR8Wz.gif]

**Comment by salimhb** [comment]
I thought that with paymill, paypal would be just another payment method and we don't need any special case for paypal, right?

**Comment by koos** [comment]
Yeah it is bit the process is still paypalish so we need to create our donation
and supporter model after a successful paypal payment. Would be cool if someone cod have a look. Thanks

Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pi&cv=6.4.27&pv=8.3]
On Mo., Juli 6, 2015 at 5:36 nachm., salimhb <notifications@github.com> wrote:
I thought that with paymill, paypal would be just another payment method and we
don't need any special case for paypal, right?

—
Reply to this email directly or view it on GitHub
[https://github.com/railslove/startups-fuer-fortuna.github.io/issues/18#issuecomment-118897611] .[https://github.com/notifications/beacon/AAAgl5A2JStS1w0PsyYpCxIYhAwaWFljks5oapgbgaJpZM4FR8Wz.gif]
